shell/browser/api/atom_api_web_contents.cc

```diff
@@ -399,7 +399,7 @@ WebContents::WebContents(v8::Isolate* isolate,
   // Whether to enable DevTools.
   options.Get("devTools", &enable_devtools_);
 
-  bool initially_shown = true;
+  bool initially_shown = type_ != Type::BROWSER_VIEW;
```

Comment:
We should have a comment explaining the purpose of this line. A link to this PR should probably be enough.

Reply:
Done!